.github/workflows/integrate.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
           channel: ${{ matrix.microk8s-versions }}
           microk8s-addons: "dns hostpath-storage rbac metallb:10.64.140.43-10.64.140.49"
           juju-channel: 3.1/stable
-          charmcraft-channel: latest/edge
+          charmcraft-channel: latest/candidate
```

Comment:
Maybe we can link the issue in the comment

Reply:
I like putting this to a more stable version, but I don't like that we're doing it in one repo.  This should be across all repos, or have a linked issue like @misohu mentioned saying why it is different from the others and when we'll switch it back.

Reply:
In other repositories we have pinned it to `latest/candidate`, though. I think this repo is one of the few (if not the only one) where we use edge.

Reply:
I was just wondering like If I am first time reading this code I would be wondering why is this candidate why not stable :)

Reply:
@misohu gotcha, a while ago the team agreed to pin the `latest/candidate` to receive the latest changes in an "almost stable" channel without having to wait for a full release cycle, preventing our CI to break. The change in this repository is to align to most of our other repositories, so if we decide to go with something other than latest/candidate, we can start the conversation.

Reply:
Resolving this because we talked offline and are in agreement since our other repos use latest/candidate.